change/@uifabric-charting-2019-11-21-12-59-29-user-v-sivsar-flickeringLegendChartIssue.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "comment": "removing the sticky={true} property from the hover card as the card should hide when the mouse goes away from the target element, and also added the gap space 12 to avoid the flickering issue of the hover card",
+  "packageName": "@uifabric/charting",
+  "email": "v-sivsar@microsoft.com",
+  "commit": "b37e7aeba72e73d45e239558a08f8636980e1630",
+  "date": "2019-11-21T07:29:29.154Z",
+  "file": "C:\\git\\office-ui-fabric\\change\\@uifabric-charting-2019-11-21-12-59-29-user-v-sivsar-flickeringLegendChartIssue.json"
```

Comment:
We've updated beachball version to not generate this line of code. Two things:
1. make sure you delete this line in this PR
2. merge with master and run yarn to get the latest version of the tool

Reply:
i din't understood, which line i should remove?  can you specify the line numbers

Reply:
After updating beachball, line 8 (the `file` attribute) should no longer appear.